SpatialGDK/Documentation/content/glossary.md

```diff
@@ -1,33 +1,119 @@
 # Glossary
+This glossary covers:
+* Terms used in this documentation
+* Terms relevant to the GDK for Unreal
+* SpatialOS terms used in the GDK for Unreal documentation
 
-## SpatialOS terms
-See the [SpatialOS glossary](https://docs.improbable.io/reference/latest/shared/glossary) for terms specific to SpatialOS.
+**SpatialOS documentation**<br/>
+Many SpatialOS term definitions link to further information in the [SpatialOS documentation](https://docs.improbable.io/reference/latest/index). This documentation covers the SpatialOS SDK - a software development kit for SpatialOS which underpins the GDK for Unreal, as well some GDK-relevant tools; the [Console](#console), [schema](#schema), [snapshots](#snapshots), and [configuration files](#configuration-files) in particular.
+
+Note that this SpatialOS documentation assumes you are developing a SpatialOS game using the SpatialOS SDKs ([The Worker SDK and Platform SDK](#https://docs.improbable.io/reference/latest/shared/get-started/working-with-spatialos)), so it may reference content relevant to that workflow only. While the SpatialOS SDK underpins the GDK for Unreal, the two workflows are not always the same.
 
-## GDK for Unreal repositories
-* The SpatialOS GDK for Unreal: https://github.com/spatialos/UnrealGDK
-* Starter Project: https://github.com/spatialos/UnrealGDKStarterProject
-* SpatialOS GDK for Unreal fork of Unreal Engine: https://github.com/improbableio/UnrealEngine/tree/4.20-SpatialOSUnrealGDK<br>
-You may get a 404 error from this link. To get access, see [these instructions]({{urlRoot}}/setup-and-installing#unreal-engine-eula).
-* Third-Person Shooter: https://github.com/spatialos/UnrealGDKThirdPersonShooter
-* Test Suite: https://github.com/spatialos/UnrealGDKTestSuite
+## GDK for Unreal documentation terms
+* `<GameRoot>` - The folder containing your game project's `.uproject` and source folder.  
+* `<ProjectRoot>` - The folder containing your `<GameRoot>`.  
+* `<YourProject>` - Name of your game project's `.uproject` (for example, `\<GameRoot>\StarterProject.uproject`).
 
 ## GDK for Unreal terms
 
 ### Actor handover
- Handover is a new `UPROPERTY` tag. It allows games built in Unreal which uses single-server architecture to take advantage of SpatialOS’ distributed, persistent server architecture. See [Actor and entity property handover between server-workers]({{urlRoot}}/content/handover-between-server-workers.md).
+Handover is a new `UPROPERTY` tag. It allows games built in Unreal which uses single-server architecture to take advantage of SpatialOS’ distributed, persistent server architecture. See [Actor and entity property handover between server-workers]({{urlRoot}}/content/handover-between-server-workers.md).
```

Comment:
```suggestion
Handover is a new `UPROPERTY` tag. It allows games built in Unreal which use single-server architecture to take advantage of SpatialOS’ distributed, persistent server architecture. See [Actor and entity property handover between server-workers]({{urlRoot}}/content/handover-between-server-workers.md).
```

(Assuming that "games" is the subject of the "which", rather than "Unreal" being the subject of "which", and so "use" should agree with "games".)